packages/now-cli/src/commands/dns/add.ts

```diff
@@ -54,9 +55,34 @@ export default async function add(
     );
     return 1;
   }
+ 
+  const { domain, data: argData } = parsedParams;
+  if (argData && argData.name.endsWith(domain)) {
+    output.log(`${chalk.yellow('Warning!')} Domain identified inside the subdomain argument. The following record will be created:`);
+    switch (argData.type) {
+      case 'MX':
+        output.log(
+          chalk.bold(`${argData.type} ${argData.value} ${argData.mxPriority}`)
+        );
+        break;
+      case 'SRV':
+        output.log(
+          chalk.bold(`${argData.type} ${argData.srv.priority} ${argData.srv.port} ${argData.srv.weight} ${argData.srv.target}`)
+        );
+        break;
+      default:
+        output.log(
+          chalk.bold(`${argData.type} ${argData.value}`)
+        );
+        break;
+    }
+    if (!await promptBool(`Do you want to create this record on "${chalk.bold(argData.name)}.${chalk.bold(domain)}?"`)) {
```

Comment:
```suggestion
    if (!(await promptBool(`Do you want to create this record on "${chalk.bold(argData.name)}.${chalk.bold(domain))}?"`)) {
```

Reply:
Do we also have a `skip` flag like `-y` for this to make sure it works without user interaction?

Reply:
Nice catch! I will add that...